examples/dishwasher-app/silabs/data_model/dishwasher-thread-app.matter

```diff
@@ -2470,43 +2472,25 @@ endpoint 0 {
   server cluster NetworkCommissioning {
     callback attribute maxNetworks;
     callback attribute networks;
-    callback attribute scanMaxTimeSeconds;
-    callback attribute connectMaxTimeSeconds;
     callback attribute interfaceEnabled;
     callback attribute lastNetworkingStatus;
     callback attribute lastNetworkID;
     callback attribute lastConnectErrorValue;
-    callback attribute supportedThreadFeatures;
-    callback attribute threadVersion;
+    callback attribute generatedCommandList;
+    callback attribute acceptedCommandList;
+    callback attribute attributeList;
     callback attribute featureMap;
     callback attribute clusterRevision;
-
-    handle command ScanNetworks;
-    handle command ScanNetworksResponse;
-    handle command AddOrUpdateWiFiNetwork;
-    handle command AddOrUpdateThreadNetwork;
-    handle command RemoveNetwork;
-    handle command NetworkConfigResponse;
-    handle command ConnectNetwork;
-    handle command ConnectNetworkResponse;
-    handle command ReorderNetwork;
-  }
-
-  server cluster DiagnosticLogs {
-    ram      attribute featureMap default = 0;
-    ram      attribute clusterRevision default = 1;
-
-    handle command RetrieveLogsRequest;
   }
 
   server cluster GeneralDiagnostics {
-    emits event BootReason;
```

Comment:
I see we are removing support for BootReason, we have an objective to fix that today so I would rather not remove it.